.github/workflows/golang.yaml

```diff
@@ -22,30 +22,33 @@ on:
     branches: [ main, develop ]
 
 env:
+  GO_MODULE: github.com/finleap-connect/monoctl
   GO_VERSION: 1.18.x
-  GINKGO_VERSION: v1.16.4
+  GINKGO_VERSION: v1.16.5
+  GO_CI_LINT_VERSION: v1.46.1
 
 jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
       - name: Install Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
           go-version: ${{ env.GO_VERSION }}
       - name: Checkout code
         uses: actions/checkout@v3
       - name: Run linters
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.45.2
+          version: ${{ env.GO_CI_LINT_VERSION }}
           args: --timeout=5m
   test:
     needs: lint
     runs-on: ubuntu-latest
     steps:
       - name: Install Go
-        uses: actions/setup-go@v2
+        if: success()
```

Comment:
I believe this is redundant as needs imply the same behavior as well and the test job will run if and only if the lint job has successfully finished. See [jobs.<job_id>.needs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds)